samples/snippets/pom.xml

```diff
@@ -43,4 +43,36 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+<profiles>
+
+  <profile>
+    <id>TranslateFromMic</id>
```

Comment:
What's this for?

Reply:
I was testing something and forgot to delete. REMOVED